Wabbajack/AppState.cs

```diff
@@ -315,7 +316,46 @@ public string SplashScreenSummary
 
         public void OnPropertyChanged(string name)
         {
-            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(name));
+            if(PropertyChanged != null)
+            {
+                PropertyChanged(this, new PropertyChangedEventArgs(name));
+            }
+        }
+        public string Error
```

Comment:
Is this property being used anywhere?

Reply:
It's required by the IDataErrorInfo interface.